modules/data-model/forms/impl/src/main/java/io/uhndata/cards/forms/internal/serialize/AbstractFormToStringSerializer.java

```diff
@@ -326,4 +345,47 @@ private String getSectionInstanceSuffix(final JsonObject answerSectionJson,
     abstract void formatPedigree(String value, StringBuilder result);
 
     abstract void formatNote(String note, StringBuilder result);
+
+    class DefinitionComparator implements Comparator<JsonObject>
+    {
+        private final List<String> definitionUuids;
+
+        DefinitionComparator(final JsonObject definition)
+        {
+            this.definitionUuids = definition.values().stream()
+                .filter(value -> ValueType.OBJECT.equals(value.getValueType()))
+                .map(JsonValue::asJsonObject)
+                .filter(value -> value.containsKey(PRIMARY_TYPE_KEY)
+                    && ("cards:Section".equals(value.getString(PRIMARY_TYPE_KEY))
+                    || "cards:Question".equals(value.getString(PRIMARY_TYPE_KEY))))
```

Comment:
I would do this instead:

```suggestion
                .filter(value -> value.containsKey(PRIMARY_TYPE_KEY))
                .filter(value -> value.containsKey(UUID_KEY))
```